modules/stochastic_tools/include/utils/AdaptiveMonteCarloUtils.h

```diff
@@ -10,6 +10,7 @@
 #pragma once
 
 #include "MooseUtils.h"
+#include "Sampler.h"
```

Comment:
I think you need to include Distribution, no Sampler
```suggestion
#include "Distribution.h"
```

Reply:
That's true! Thanks